### PR TITLE
updated JL MD

### LINKED ISSUE #16: Add Jupyter notebook comments
Add further comments to highlight learning points on each exercise.

add additional MD where appropriate

### PR DESCRIPTION
**Pull request by j-emberton**
Fleshed out Jupyter notebook comments

closes #16

### CODE REVIEW

File: presentation/images/.DS_Store
Comment (jatkinson1000) [inline presentation/images/.DS_Store:1]: Can we get rid of this `.DS_Store`?

File: notebook/Scientific_Visualisation.ipynb
Comment (jatkinson1000) [inline notebook/Scientific_Visualisation.ipynb:1]: This looks like it has all of the output from running the cells creating a very large diff.
Please could you do `Kernel -> Restart Kernel and Clear Outputs of All Cells` and then save and check in that version of the notebook please?

**Review by jatkinson1000 — CHANGES_REQUESTED** [review]
Thanks @j-emberton 

This generally feels sensible, but please could you remove the rogue `.DS_Store` and clean the notebook before checking in.

If we both check in clean notebooks that will help us avoid merge conflicts when working on it asynchronously

**Comment by j-emberton** [comment]
should all be done

**Review by jatkinson1000 — APPROVED** [review]
Thanks for addressing that @j-emberton 
This looks good to me now so I will merge shortly.

**Pull request merged**